dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -6,7 +6,7 @@ logstashCollector=Logstash Collector
 logstashCollector.desc=Logstash collector gathers data from various sources and forwards the data to a logstash server using Lumberjack protocol.
 
 source=Source
-source.desc=Specifies a source to be used by the logstash collector.
+source.desc=Specifies a list of comma-separated sources that are used by the logstash collector. Valid values are message, trace, accessLog, ffdc, garbageCollection, and audit. To use the audit source, enable the Liberty audit feature.
```

Comment:
it's odd that we say to enable the audit feature to get audit events, but we don't mention that you need an accessLogging element if you want accessLog events.  I'd suggest adding a sentence to cover that (and also adding that to the descriptions for the messageSource / consoleSource of logging)